inner-scripts/kubestellar

```diff
@@ -38,54 +38,98 @@ remove=0
 cleanup=0
 espw_name="espw"
 log_folder=$(pwd)/kubestellar-logs
-local_kcp=true
 imws="root:imw1"
 wmws="root:wmw1"
 
 function echoerr() {
    echo "ERROR: $1" >&2
 }
 
+function create_space() { # usage: space_name
```

Comment:
This should be a script in its own right.

Reply:
Separated into its own script.